aion/test/models/room_test.exs

```diff
@@ -0,0 +1,18 @@
+defmodule Aion.RoomTest do
+  use Aion.ModelCase
+
+  alias Aion.Room
+
+  @valid_attrs %{description: "some content", name: "some content"}
+  @invalid_attrs %{}
+
+  test "changeset with valid attributes" do
+    changeset = Room.changeset(%Room{}, @valid_attrs)
+    assert changeset.valid?
+  end
+
+  test "changeset with invalid attributes" do
+    changeset = Room.changeset(%Room{}, @invalid_attrs)
+    refute changeset.valid?
```

Comment:
That's good, we may change the model at some point to a model that would make this test fail, it's good to get an alert that things that we say 'are always true' are actually not after a change.